src/desktop/assets/misc.styl

```diff
@@ -1,71 +1,56 @@
 // authentication
-@require '../apps/authentication/reset_password.styl'
-
+@require '../apps/authentication/reset_password.styl';
 // about
-@require '../components/jump/index.styl'
-@require '../apps/about/stylesheets'
-
+@require '../components/jump/index.styl';
+@require '../apps/about/stylesheets';
 // contact
-@require '../components/chevron_list'
-@require '../components/multi_page_modal'
-@require '../apps/contact/stylesheets'
-
+@require '../components/chevron_list';
+@require '../components/multi_page_modal';
+@require '../apps/contact/stylesheets';
 // feature
-@require '../components/tooltips'
-@require '../components/artwork_columns'
-@require '../components/artwork_item/stylesheets'
-@require '../components/auction_artworks/stylesheets'
-@require '../components/clock'
-@require '../apps/feature'
-
+@require '../components/tooltips';
+@require '../components/artwork_columns';
+@require '../components/artwork_item/stylesheets';
+@require '../components/auction_artworks/stylesheets';
+@require '../components/clock';
+@require '../apps/feature';
 // jobs
-@require '../components/chevron_list'
-@require '../apps/jobs/stylesheets'
-
+@require '../components/chevron_list';
+@require '../apps/jobs/stylesheets';
 // page
-@require '../apps/page'
-
+@require '../apps/page';
 // press
-@require '../components/chevron_list'
-@require '../apps/press/stylesheets'
-
+@require '../components/chevron_list';
+@require '../apps/press/stylesheets';
 // static
-@require '../apps/static/stylesheets'
-
+@require '../apps/static/stylesheets';
 // style_guide
-@require '../components/artwork_columns'
-@require '../components/artwork_item/stylesheets'
-@require '../components/fillwidth_row'
-@require '../components/profile_badge'
-@require '../components/tooltips'
-@require '../components/partner_buttons'
-@require '../components/grid'
-@require '../components/bordered_pulldown'
-@require '../components/toggle_list'
-@require '../components/popover'
-@require '../components/hover_pulldown'
-@require '../components/chevron_list'
-@require '../components/clock'
-@require '../components/multi_page_modal'
-@require '../apps/style_guide'
```

Comment:
This line is the only significant change in this file, yeah? Was a little hard to see bc of the heavy linting — is that something we enforce with Stylus files now?

Reply:
Yep. 😓. I think that was actually due to prettier with my editor. 

It'd be good to do a prettier pass on all these files at some point..